CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/home/HomeViewModel.kt

```diff
@@ -15,7 +15,7 @@ import javax.inject.Inject
 
 interface Output {
 
-    val users: StateFlow<List<UserEntity>>
+    val users: StateFlow<List<User>>
```

Comment:
We're supposed to have a different model for the `presentation` layer. We map the `domain` model to the `presentation` model inside `ViewModel`s. How about we do it in this PR? 💡 
```suggestion
    val users: StateFlow<List<UserUiModel>>
```

Reply:
@AVI5HEK Honestly, I did not see any benefit of this mapping, could you please explain more use cases in real project that we need this step? 🙏